frontend/src/components/Header.vue

```diff
@@ -1,5 +1,5 @@
 <template>
-  <v-system-bar absolute color="rgb(0, 51, 102)" height="66rem" class="sysBar">
+  <v-system-bar absolute color="rgb(0, 51, 102)" class="sysBar" :style="{ height: '66px', textAlign: 'left' }" >
```

Comment:
Can we just move these styles into sysBar class?

Reply:
Moved inline style to sysBar class